### PR TITLE
PR: Generate PyPI long description from README.md

### PR DESCRIPTION
**Pull request by doudz**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,37 +13,15 @@
...
-.. code-block:: python
-
-    # Get icons by name.
-    fa_icon = qta.icon('fa5s.flag')
-    fa_button = QtGui.QPushButton(fa_icon, 'Font Awesome!')
-
-    asl_icon = qta.icon('ei.asl')
-    elusive_button = QtGui.QPushButton(asl_icon, 'Elusive Icons!')
-"""
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
```
Comment (ccordoba12) [inline setup.py]: Please change `path.join` to `os.path.join` here. That's why our tests are failing.

**Comment by doudz** [comment]
oops, fixed !

File: setup.py (hunk 1 above)
Comment (doudz) [reply setup.py]: fixed

File: setup.py [hunk 2]
```diff
@@ -13,37 +13,15 @@
...
-.. code-block:: python
-
-    # Get icons by name.
-    fa_icon = qta.icon('fa5s.flag')
-    fa_button = QtGui.QPushButton(fa_icon, 'Font Awesome!')
-
-    asl_icon = qta.icon('ei.asl')
-    elusive_button = QtGui.QPushButton(asl_icon, 'Elusive Icons!')
-"""
+with open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
```
Comment (ccordoba12) [inline setup.py]: `encoding` is not a valid keyword for `open` in Python 2. I think you can use `io.open` instead.

**Pull request merged**